Task3/js/main.js

```diff
@@ -0,0 +1,45 @@
+//1.
+//listen for window resize event
+window.addEventListener('resize', function(event){
+  var newWidth = window.innerWidth;
+  var newHeight = window.innerHeight;
+  console.log(newWidth)
+  console.log(newHeight)
+});
+
+//2.
+//Change URL params
+// Construct URLSearchParams object instance from current URL querystring.
+function setUrlParameter(param_name, param_value){
```

Comment:
This method doesn't work correctly on `localhost` domain. You should avoid specifying absolute url.